docs/source/changelog.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 Development version
 -------------------
 
+- Extend OARCluster implementation to let OAR take into account the memory parameter (:pr:`598`, :pr:`595`)
```

Comment:
@ychiat35 I've committed the change directly to the PR. What I meant is to move the bullet point you added to the section named `Development version`. This section will be renamed to whatever the version is called upon release.

Reply:
Brief explainer: Each PR is meant to add an item to the dev-version section of the change log which links and describes the changes in the PR. Then, when the core team decides on a release, they choose a version number (loosely based on [semver](https://semver.org/lang/de/)) taking into account all changes since the last release, add a section header reflecting this version number, and leave the `Development version` section empty until a new PR on top of the released version adds items again.

Reply:
Thanks for your explanation! I think that I haven't opened my eyes enough to have perfectly missed the `Development version` part.. 

By the way, I don't really understand what the links on the PRs are for in the changelog, e.g. [:pr:`586`] gives the link https://github.com/dask/dask-jobqueue/blob/main/docs/source/changelog.rst#id4. Are the links generated somehow?

Reply:
The correct links are generated on the documentation, you can see the documentation generated from this PR [here](https://dask-jobqueue--602.org.readthedocs.build/en/602/changelog.html) and the links look fine.

You can find the link in the CI statuses (at the bottom of the page)

![image](https://user-images.githubusercontent.com/1680079/211588260-ce2b7d9b-3578-44c9-aa96-393533dc3451.png)

Reply:
Ok I see! Thank you very much for your screenshot and explanations :)